### PR TITLE
 added windows 11 , added tpm link and added one column

### PR DESCRIPTION
**Pull request by RAJU2529**
as per user feedback #9853, so i added windows 11 , added tpm link and added one column

### CODE REVIEW

File: windows/security/information-protection/tpm/trusted-platform-module-overview.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ This topic for the IT professional describes the Trusted Platform Module (TPM) a
 
 ## Feature description
 
-Trusted Platform Module (TPM) technology is designed to provide hardware-based, security-related functions. A TPM chip is a secure crypto-processor that is designed to carry out cryptographic operations. The chip includes multiple physical security mechanisms to make it tamper resistant, and malicious software is unable to tamper with the security functions of the TPM. Some of the key advantages of using TPM technology are that you can:
+[Trusted Platform Module (TPM)](https://docs.microsoft.com/windows/security/information-protection/tpm/trusted-platform-module-top-node) technology is designed to provide hardware-based, security-related functions. A TPM chip is a secure crypto-processor that is designed to carry out cryptographic operations. The chip includes multiple physical security mechanisms to make it tamper resistant, and malicious software is unable to tamper with the security functions of the TPM. Some of the key advantages of using TPM technology are that you can:
```
Comment (JohanFreelancer9) [inline windows/security/information-protection/tpm/trusted-platform-module-overview.md:31]:
```suggestion
[Trusted Platform Module (TPM)](/windows/security/information-protection/tpm/trusted-platform-module-top-node) technology is designed to provide hardware-based, security-related functions. A TPM chip is a secure crypto-processor that is designed to carry out cryptographic operations. The chip includes multiple physical security mechanisms to make it tamper-resistant, and malicious software is unable to tamper with the security functions of the TPM. Some of the key advantages of using TPM technology are that you can:
```

File: windows/security/information-protection/tpm/trusted-platform-module-overview.md [hunk 2]
```diff
@@ -75,14 +75,14 @@ Some things that you can check on the device are:
 -   Is SecureBoot supported and enabled?
 
 > [!NOTE]
->  Windows 10, Windows Server 2016 and Windows Server 2019 support Device Health Attestation with TPM 2.0. Support for TPM 1.2 was added beginning with Windows version 1607 (RS1). TPM 2.0 requires UEFI firmware. A computer with legacy BIOS and TPM 2.0 won't work as expected.
+>  Windows 11, Windows 10, Windows Server 2016 and Windows Server 2019 support Device Health Attestation with TPM 2.0. Support for TPM 1.2 was added beginning with Windows version 1607 (RS1). TPM 2.0 requires UEFI firmware. A computer with legacy BIOS and TPM 2.0 won't work as expected.
```
Comment (JohanFreelancer9) [inline windows/security/information-protection/tpm/trusted-platform-module-overview.md:78]:
```suggestion
>  Windows 11, Windows 10, Windows Server 2016, and Windows Server 2019 support Device Health Attestation with TPM 2.0. Support for TPM 1.2 was added beginning with Windows version 1607 (RS1). TPM 2.0 requires UEFI firmware. A computer with legacy BIOS and TPM 2.0 won't work as expected.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by RAJU2529** [comment]
This PR is final, not draft.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by e0i** [comment]
@tiburd

**Review by tiburd — APPROVED** [review]

**Pull request merged**